src/vfb_query_builder/test/query_roller_tests.py

```diff
@@ -52,6 +52,53 @@ def get_edits_string(old, new):
                             break
         print('New vfb.xmi')
         print(''.join(xmi))
-                            
+
+        queries = json.loads(term_info_export(escape='json'))
+        # extract out the cypher queries from the VFBconnect master branch
+        ecore=urllib.request.urlopen("https://github.com/VirtualFlyBrain/VFB_connect/raw/master/src/vfb_connect/resources/VFB_TermInfo_queries.json")
+        model = []
+        xmi = []
+        for line in ecore:
+            xmi.append(line.decode('utf-8'))
+            if "Get JSON for " in line.decode('utf-8'):
+                model.append(line.decode('utf-8'))
+        for query in queries:
+            print(query)
+            print(newquery(queries[query]))
+            for cypher in model:
+                if query in cypher:
+                    print(oldquery(cypher))
+                    print(get_edits_string(cypher, queries[query]))
+                    for index, line in enumerate(xmi):
+                        if cypher in line:
+                            xmi[index] = line.replace(cypher, queries[query])
+                            break
+        print('New VFB_TermInfo_queries.json')
+        print(''.join(xmi))
+
+        queries = json.loads(single_input_export(escape='json'))
+        # extract out the cypher queries from the VFBconnect master branch
+        ecore=urllib.request.urlopen("https://github.com/VirtualFlyBrain/VFB_connect/raw/master/src/vfb_connect/resources/VFB_results_single_input.json")
```

Comment:
function q_name starts with `Get JSON for `, but in VFB_results_single_input.json we don't have this prefix